recognition.py

```diff
@@ -0,0 +1,127 @@
+import cv2
+import numpy as np
+import copy
+import math
+
+# parameters
+capture_region_x=0.5  # roi x start point
+capture_region_y=0.8  # roi y start point
+threshold = 60  #  threshold
+blur_value = 41  # GaussianBlur parameter
```

Comment:
did you try using different values of sigma for Gaussian Blur? in my experience, different values of sigma can have different effects on the edge blur. Also try bilateral blur.

Reply:
Yeah we can mess around with this a bit....there's also user thresholding built-in